lms/templates/page_banner.html

```diff
@@ -0,0 +1,24 @@
+## mako
+
+<%page expression_filter="h"/>
+
+<%namespace name='static' file='static_content.html'/>
+<% online_help_token = self.online_help_token() if hasattr(self, 'online_help_token') else None %>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+from openedx.core.djangoapps.util.user_messages import user_messages
+%>
+
+% if user_messages:
+    <div class="page-banner">
+        <div class="user-messages">
+            % for message in user_messages(request):
+                <div class="alert ${HTML(message.css_class)}" role="alert">
```

Comment:
Isn't that handled by the Mako default escaping rule? Also, these classes are generated by my code, so it is very low risk of any XSS.